shared/src/test/scala/mlscript/DiffTests.scala

```diff
@@ -276,6 +282,9 @@ class DiffTests
               case l :: _ => out.println(l)
             }
             return ()
+          case "UseIR" => mode.copy(useIR = true)
+          case "InterpIR" => mode.copy(interpIR = true)
```

Comment:
By convention, block-commands start with a small letter:
```suggestion
          case "interpIR" => mode.copy(interpIR = true)
```